taccsite_cms/templates/guides/rsync_data_transfer.html

```diff
@@ -0,0 +1,510 @@
+      </p>
+      <blockquote>
+        <p>
+          Note: It is possible to use these command line tools if your local machine runs Windows, but you will need to use a ssh client (ex.
+          <a target="_blank" href="https://www.putty.org/">PuTTY</a>
+          ).
+        </p>
+        <p>
+          However, for the scope of this project and ease in data transfer, it is recommended that Windows users follow our
+          <a href="#cyberduck_transfer">Transfer Data with Cyberduck</a>
+          guide below.
+        </p>
+      </blockquote>
+      <p>
+        For users that are new to the command line, using either <code>scp</code> or <code>sftp</code> to transfer data is advised.
+      </p>
+      <h4 id="data_transfer_prereqs">
+        Prerequisites for Data Transfer with Command Line Tools
+      </h4>
+
+      <p>Before we begin, you will need to know:</p>
+
+      <ul>
+        <li>the path to your data file(s) on your local system</li>
+        <li>the path to your transfer directory on the remote storage server</li>
+      </ul>
+
+      <h4 id="local_data_path">
+        Determining the Path to Your Data File(s) on Your Local System
+      </h4>
+
+      <p>
+        In order to transfer your project data, you will need to know where your file(s) is located on your local system. To do so, navigate through the file system on your local computer.
+      </p>
+      <p>
+        For example, you can do so on Mac by going to Finder. Common data file locations such as Desktop and Documents should be accessible through the Favorites pane on the left-hand side of the window. You can also do Finder &gt; Go to find other common locations on your computer.
+      </p>
+      <p>
+        Once you have navigated to where your file(s) is located, take note of its path by right-clicking on the file and selecting Get Info.
+      </p>
+      <figure id="figure1">
+        <img src="https://a2cps-staging.tacc.utexas.edu/media/filer_public_thumbnails/filer_public/6c/ea/6ceacba9-7824-4925-b924-dd317040b629/picture1.png__1170x0_q85_subsampling-2_upscale.png" />
+        <figcaption>
+          Figure 1. Use Get Info to determine “Where” the path of your data file(s) is
+        </figcaption>
+      </figure>
+
+      <p>
+        For example, a file located in an <code>portal-data</code> folder in
+        <code>Documents</code> would have the following path:
+      </p>
+      <p>
+        <code>/Users/myname/Documents/portal-data/my_file.txt</code>
+      </p>
+      <h4 id="storage_data_path">
+        Determining the Path to Your Transfer Directory
+      </h4>
+
+      <p>
+        A transfer directory on the remote storage server has already been set up for you and is available for data transfer. The transfer directory path is individualized to your research center/institution.
+      </p>
+      <details class="small"><summary>Examples:</summary>
+        <pre><code>/corral-secure/projects/A2CPS/submissions/utaustin/</code></pre>
```

Comment:
<pre><code>/corral-repl/utexas/project_name</code></pre>
<pre><code>/corral-repl/uta/project_name</code></pre>
<pre><code>/corral-repl/utep/project_name</code></pre>
<pre><code>/corral-repl/uth/project_name</code></pre>

Reply:
These may be setup differently on corral-secure, I am not familiar with that area of the system yet.

Reply:
most of the portal projects, when they have their own location outside of `/corral-repl/tacc/aci`, have been under `/corral-repl/projects/[TAS-project-name]`, for non-secure projects.  the path for corral-secure for A2CPS sounds correct to me too, since we're using a similar path for our test secure location for that portal.

My question is then: are we trying to include paths for other locations in these guides, or make it more portal related?  I've never heard of any other paths on /corral-repl besides /tacc and /projects, so I don't know how often those are used or if they still are.

Reply:
(sigh) github showed me two comments submitted on this so I deleted one and it looks like it only actually added one, so here's my attempt at recreating it, potentially shorter....

I've never heard of the uta/utexas/utep/uth paths on corral.  Everything I've seen has been either under tacc or projects.  The corral-secure path for A2CPS is correct as far as I'm aware; it matches the path style of `/corral-secure/projects/[TAS-Project-Name]` that we are using for a test secure location for the Data Depot feature.

Does this guide need to be able to handle those other paths, if they're still being used?  or are we focusing on what path styles we've seen for portals?

Reply:
@taoteg, I will add your examples.

@SilversunKSauri, thanks.

This guide now just provides examples, where helpful. I will retain this A2CPS path as is for this example. I believe it is accurate because I see [a confident repeat use of the path in a Slack message](https://tacc-team.slack.com/archives/CNG25CWLW/p1612463263081700?thread_ts=1612460517.075900&cid=CNG25CWLW).

<details><summary>P.S. Possible explanation of the GitHub UI trouble you had…</summary>

> You may have been seeing a "Pending" comment of an unfinished [review](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-request-reviews#about-pull-request-reviews) and/or [a standalone comment on a PR](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request); and either may have been an "in-memory" comment you wrote but never added (GitHub remembers those for you and restores them upon return to, or reload of, page).

</details>

Reply:
1. @jgentle, are these the intended examples (append `/submissions/utaustin/`)?
    - `/corral-repl/utexas/`__`project_name`__`/submissions/utaustin/`
    - `/corral-repl/uta/`__`project_name`__`/submissions/utaustin/`
    - `/corral-repl/utep/`__`project_name`__`/submissions/utaustin/`
    - `/corral-repl/uth/`__`project_name`__`/submissions/utaustin/`

2. Anyone, is the `/submissions/utaustin/` unnecessary for this example, such that I should change the A2CPS example to `/corral-secure/projects/A2CPS/`?

Reply:
I do not think they would be included as examples. 

I DO know that the portals have previously defaulted to this location for all Project folders: 
`/corral-repl/tacc/aci/{project_name}/projects`
with some (such as 3dem and SD2E) having entirely unique locations that differ from all these examples.

My point here is that the secure data area is entirely different than the regular projects data area.

I am not up to speed on CORE-v2 enough to know if going forward it is intended that ALL Projects will default to a secure corral location or if we will be spanning both. I would not assume they will default top secure-corral because there is aditionall overhead in setting up and accessing those locations, but I could be wrong.

Someone on the v2 team who groks the MANY locations of our portal projects folders for v1 and v2 (and any changes that migration from v1 to v2 might entail) needs to answer this.

I just want to ensure that the guide is generic enough to account for variations in the possible project folder paths.

Reply:
I have fixed my misunderstanding of the examples in my previous comment's question to John.

But, I don't want to add new examples unless they are real paths or confirmed patterns.

Reply:
I am not sure.
I do not grok the `utaustin` coming after the project submission folder. That may be an aspect of A2COPS being a multi-institution effort.
Our previous portals had no concept of this in their paths.
Examples from corral:
`/corral-repl/tacc/aci/A2CPS/projects/A2CPS-##`
`/corral-repl/tacc/aci/UTRC/projects/UTRC-##`
`/corral-repl/tacc/aci/COVID19/projects/COVID19-##`
`/corral-repl/tacc/aci/PT2050/projects/PT2050-##`
`/corral-repl/tacc/aci/CII/projects/CII-##`

There may be some overloaded use of the term "Projects" going on here. I am referring to a Project (ie.g., a folder created through the portal interface) that results in one of the above folders being created in the locations defined above.

The concept of submission folders used for other data ingest mechanisms (e.g., rsync, scp, sftp, globus) may not be talking about the same concept.

Someone on v2 team needs to clarify the delta here and identify specifically what we are referring to in these guides. My understanding is limited to what I have just added.

Reply:
@rstijerina Please check the "Examples:" and the paragraph before.

1. Is it "portal-agnostic" enough?
2. Do you know any example paths that should be added?

Reply:
This file was renamed. This thread of discussion has been restarted as a new comment on the file under its new name.

See: https://github.com/TACC/Core-CMS/pull/152/files#r598958610